site/content/blog/2021-03-12-report-competition/index.md

```diff
@@ -0,0 +1,145 @@
+
+If you need more advice and guidance then ask on the [ZAP User Group](https://groups.google.com/g/zaproxy-users).
+
+[Thymeleaf](https://www.thymeleaf.org/) (the templating engine which we are using) is well documented but if you do have problems with it then ask on the ZAP User Group - we do now have some experience with it.
+
+Also join the User Group for updates on the Report Generation add-on especially for features related to this competition.
+
+To submit a report to this competition raise a pull request adding your report in a new directory underneath 
+[reports/src/main/zapHomeFiles/reports](https://github.com/zaproxy/zap-extensions/tree/master/addOns/reports/src/main/zapHomeFiles/reports) - 
+if you are not sure how to do that ask on the User group as well.
+
+We are **really** looking forward to seeing what you produce - lets make one of ZAP's traditional weaknesses into one of its strengths!
+
+## What do people want from reports?
+
+We recently published a questionnaire asking this very question.
+
+The relevant answers are shown below, but for me the highlights are that people want:
+
+* Good look and feel
+* Charts
+* Ability to customise
+* More data available
+
+Its also worth checking through previous posts to the [ZAP User Group](https://groups.google.com/g/zaproxy-users).
+
+One request that keeps coming up is the ability to include details of the scan rules that have passed.
+This is not possible right now but thats one of the changes we will be making so we will be keen to see this supported in the new templates.
+
+In the answers below the higher the number the more important it was for people.
+
+#### How important is the look and feel of the report to you?
+![](./images/lookandfeel.png)
+* Not important: 6.5%
+* Less important: 6.5%
+* Average importance: 19.4%
+* More important: 29%
+* Very important: 38.7%
+
+#### How important is the ability to include charts in the report to you?
+![](./images/lookandfeel.png)
+* Not important: 9.7%
+* Less important: 12.9%
+* Average importance: 22.6%
+* More important: 22.6%
+* Very important: 32.3%
+
+#### How useful would it be for you to be able to configure the report format?
+![](./images/configure.png)
+* Not important: 0%
+* Less important: 9.7%
+* Average importance: 12.9%
+* More important: 12.9%
+* Very important: 64.5%
+
+#### How useful would it be for you to be able to include your own custom information?
+![](./images/custom.png)
+* Not important: 0%
+* Less important: 0%
+* Average importance: 22.6%
+* More important: 16.1%
+* Very important: 61.3%
+
+#### Any other reporting features you would like us to include?
+This was a free text field and the answers are included without any changes:
+
+* Be able to translate
+* Branding. Not with the intention of branding for a hired engagement, but for internal reporting. 
+* thx!
+* The support for certain report formats like Hipaa, is27001, owasp Top10, pci, .... would be really cool :-)
```

Comment:
```suggestion
* The support for certain report formats like HIPAA, ISO27001, OWASP Top10, PCI, .... would be really cool :-)
```

Reply:
I guess this is a quote, but I'd suggest at least fixing the "ISO" one.

Reply:
"This was a free text field and the answers are included without any changes" I'd rather not change anything tbh

Reply:
Ok